pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -713,8 +714,34 @@ func PodFitsResources(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *s
 	}
 
 	var podRequest *schedulercache.Resource
+	// No extended resources should be ignored by default unless specified via
+	// extender configuration.
+	ignoredExtendedResources := sets.NewString()
+	// Missing extended resources will be ignored by default unless a predicate
+	// metadata is provided.
+	//
+	// This predicate is used by scheduler, kubelet, cluster autoscaler and
```

Comment:
@dchen1107 @mwielgus 
This is a backward incompatible change in kubelet and autoscaler, but it applies only to Pods that have extended resources missing on the node. Do you guys approve such a change?